README.md

```diff
@@ -1,5 +1,56 @@
-# Galasa
-This is a home for all of the Galasa Java code.
+
+
+# Galasa Homepage
+
+[![Build and publish to github pages](https://github.com/techcobweb/galasa.dev/actions/workflows/build.yaml/badge.svg)](https://github.com/techcobweb/galasa.dev/actions/workflows/build.yaml)
```

Comment:
This link does not lead to a workflow. Should it also be pointing it your fork too?